.github/workflows/CI.yml

```
@@ -0,0 +1,54 @@
+name: SonarCloud Analysis with dotnet-coverage
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)